src/import/meson.build

```diff
@@ -52,9 +52,8 @@ if conf.get('ENABLE_IMPORTD') == 1
 endif
 
 tests += [
-        [files('test-qcow2.c',
-               'qcow2-util.c'),
-         [],
-         [libz],
-         [], 'HAVE_ZLIB', 'manual'],
+        { 'sources' : files('test-qcow2.c', 'qcow2-util.c'),
```

Comment:
No, let's not put multiple files on the same line. We were slowly changing towards a format where all files as listed on separate lines, and if there's more than one file name, a comma and newline after the last file name.